styles/widgets/generic/gridBase.generic.less

```diff
@@ -434,6 +434,10 @@
                     &.dx-@{widgetName}-group-space {
                         border-right-color: ~"@{@{widgetName}-hover-bg}";
                     }
+
+                    &:first-child {
```

Comment:
Selection column can have non-first position. Also, the first cell is not always selection column. So it will be better to use .dx-command-select selector